src/miner.cpp

```diff
@@ -239,7 +239,7 @@ std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock(const CScript& sc
         coinbaseTx.vout.resize(2);
 
         // Explicitly set miner reward
-        coinbaseTx.vout[0].nValue = CalculateCoinbaseReward(blockReward, consensus.dist.masternode);
+        coinbaseTx.vout[0].nValue = nFees + CalculateCoinbaseReward(blockReward, consensus.dist.masternode);
```

Comment:
It should be covered by hardfork hiehgt

Reply:
If we go without hardfork, newer nodes will generate bigger reward than older one, It's inconstancy.

Reply:
I will make the change in another PR based on 1.9.x